miniapps/tools/tmop-check-metric.cpp

```diff
@@ -153,8 +153,8 @@ int main(int argc, char *argv[])
    auto integ = new TMOP_Integrator(metric, &tc, NULL);
    a.AddDomainIntegrator(integ);
 
-   ElementTransformation &Tr = *mesh->GetElementTransformation(0);
-   const FiniteElement &fe = *fespace.GetFE(0);
+   ElementTransformation &Tr = *mesh->GetTypicalElementTransformation();
+   const FiniteElement &fe = *fespace.GetTypicalFE();
```

Comment:
There's no need to change the code here -- the mesh with 1 element is constructed just above.

Reply:
Reverted in 4c96ebc65280284531ef470aa895753f061b3b2f.